index.js

```diff
@@ -24,6 +42,7 @@ const main = (stream, {showCursor = false} = {}) => {
 		}
 
 		let output = args.join(' ') + '\n';
+		output = fitToTerminalHeight(stream, output);
 		const width = getWidth(stream);
 		if (output === previousOutput && previousWidth === width) {
```

Comment:
Do we need to add the height to this check?

Reply:
I think not, since it's a post-fitting comparison.  If the height changes and that causes the output to change, this comparison will catch it.